src/main/java/main/Duke.java

```diff
@@ -0,0 +1,62 @@
+package main;
+
+import parser.Parser;
+import tasks.Task;
+import ui.Ui;
+import actions.TaskList;
+import storage.Storage;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+
+    private Ui ui;
+    private Storage storage;
+    private TaskList tasklist;
+    public ArrayList<Task> taskArrayList = new ArrayList<>();
```

Comment:
I think `taskList` should already be declared in `taskArrayList` and not be accessible by `Duke` here.
Why is the access modifier for `taskArrayList` public?